README.md

```diff
@@ -30,6 +30,12 @@ To get started, sign up [here](https://clevertap.com/live-product-demo/).
 CleverTap Push Templates SDK helps you engage with your users using fancy push notification templates built specifically to work with [CleverTap Push Templates SDK](https://github.com/CleverTap/CTNotificationContent).
 Find the integration steps for the CleverTap Push Templates SDK [here](./docs/iospushtemplates.md)
 
+## ℹ️ Version Support
+| CleverTap RN SDK | React Native | Supports New Architecture? |
+|------------------|--------------|----------------------------|
+| 3.0.0+           | >=0.65       | ✅ (>=0.71)                 |
```

Comment:
@Anush-Shand Is this AND or ORed

Reply:
What if CT RN SDK is 3.0.0+ but RN is < 0.65

Reply:
It is AND

We support 3.0.0 only on 0.65+ and for new arch it should be 0.71
Technically it will work for older versions but we are setting a cap to reduce NOCs